examples/Sn_High_Pressure/command.sh

```
@@ -0,0 +1,19 @@
+# NPT
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)